lib/msal-core/tests/error/error.spec.ts

```diff
@@ -8,74 +8,105 @@ describe("Error", () => {
     const TEST_ERROR_CODE: string = "test";
     const TEST_ERROR_MSG: string = "This is a test error";
 
+    let throwFunc = (err: AuthError): void => {
```

Comment:
const

Reply:
what does this do? cant you just throw an AuthError below if all you are doing is try catching it